src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -83,24 +84,24 @@ public void execute_invalidIndexFilteredExpenditureList_throwsCommandException()
 
     @Test
     public void equals() {
-        DeleteCommand deleteFirstCommand = new DeleteCommand(INDEX_FIRST_PERSON, expenditureType);
-        DeleteCommand deleteSecondCommand = new DeleteCommand(INDEX_SECOND_PERSON, expenditureType);
+        DeleteCommand deleteFirstCommand = new DeleteCommand(INDEX_FIRST_ENTRY, expenditureType);
+        DeleteCommand deleteSecondCommand = new DeleteCommand(INDEX_SECOND_ENTRY, expenditureType);
 
         // same object -> returns true
-        assertTrue(deleteFirstCommand.equals(deleteFirstCommand));
+        assertEquals(deleteFirstCommand, deleteFirstCommand);
```

Comment:
Accepted Intellij's formatting tip about changing to `assertEquals`

Reply:
Thanks Emily!! I just changed the previous test cases that were already present in the EditCommand HAHA